wondrous-app/components/Common/Sidebar/Entity/AboutEntity/EntityMenu.tsx

```diff
@@ -0,0 +1,173 @@
+  && {
+    max-width: fit-content;
+    padding: 2px;
+    padding-right: 6px;
+    height: 32px;
+    cursor: pointer;
+    border-radius: 6px;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    background: ${({ open, theme }) => open && `${theme.palette.grey78}`};
+    :hover {
+      background: ${({ theme }) => theme.palette.grey78};
+    }
+  }
+`;
+
+export const ArrowIcon = styled((props) => (
+  <div {...props}>
+    <Arrow />
+  </div>
+))`
+  && {
+    transform: rotate(${({ open }) => (open ? `-90` : `90`)}deg);
+    display: flex;
+    height: 32px;
+    align-items: center;
+    justify-content: center;
+    svg {
+      path {
+        fill: ${({ theme }) => theme.palette.white};
+      }
+    }
+  }
+`;
+
+const MenuStyled = styled(Menu)`
+  && {
+    margin-top: 6px;
+    .MuiMenu-paper {
+      background-color: ${({ theme }) => theme.palette.grey77};
+    }
+    .MuiList-root {
+      background-color: ${({ theme }) => theme.palette.grey77};
+      min-width: 173px;
+      border-radius: 6px;
+      display: flex;
+      flex-direction: column;
+      gap: 4px;
+      padding: 4px;
+    }
+  }
+`;
+
+const Item = styled(MenuItem)`
+  && {
+    font-family: ${({ theme }) => theme.typography.fontFamily};
+    font-weight: 500;
+    font-size: 14px;
+    color: ${({ theme }) => theme.palette.white};
+    background: ${({ theme }) => theme.palette.grey87};
+    border-radius: 2px;
+    padding: 4px 9px;
+    :hover {
+      background: ${({ theme }) => theme.palette.grey58};
+    }
+  }
+`;
+
+export const NoLogoDAO = styled((props) => (
+  <div {...props}>
+    <DAOIcon
+      stroke="#fff"
+      encircled={false}
+      style={{
+        width: '28px',
+        height: '28px',
+      }}
+    />
+  </div>
+))`
+  display: flex;
+  width: 28px;
+  height: 28px;
+  border-radius: 3px;
+  align-items: center;
+  justify-content: center;
+  background: ${({ theme }) => theme.palette.grey87};
+`;
+
+export const NoLogoPod = styled((props) => (
+  <div {...props}>
+    <PodsIcon />
+  </div>
+))`
+  display: flex;
+  width: 28px;
+  height: 28px;
+  border-radius: 3px;
+  align-items: center;
+  justify-content: center;
+  background: ${({ theme }) => theme.palette.grey87}; ;
+`;
+
+const EntityMenu = ({ name, id, router, thumbnailPicture, profilePicture, canManage }) => {
+  const { orgBoard } = useBoards();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const handleClose = () => setAnchorEl(null);
+  const handleOnClickNotifications = () =>
+    router.push(orgBoard ? `/organization/settings/${id}/notifications` : `/pod/settings/${id}/notifications`);
+  const NoLogo = orgBoard ? NoLogoDAO : NoLogoPod;
+  return (
+    <>
+      <Button onClick={handleClick} open={open} disabled={!canManage}>
+        <ButtonIcon>
+          {thumbnailPicture || profilePicture ? (
+            <SafeImage
```

Comment:
there's a component for this components/Common/ProfilePictureHelpers can we adjust that / reuse it?

Reply:
I used the 'OrgProfilePicture' helper here. However, the 'PodProfilePicture' does not fit, so I used the custom one.